docs/_plugins/search_index.rb

```diff
@@ -432,6 +423,7 @@ def is_latest?(group, model)
 end
 
 Jekyll::Hooks.register :site, :post_render do |site|
+  is_incremental = site.config['incremental']
   force_reindex = editions_changed?(editions)
```

Comment:
Looks like force_reindex is not being used.
If we see a new edition, `Healthcare NLP 4.2`, for example, we should re-index all Healthcare NLP models and mark them as compatible with `Healthcare NLP 4.2`. The same situation is with models.json: we should update the `compatible_models` field.

Reply:
So, if if the edition changes, its best to do a full build. Am I right, @pabla ?

Reply:
Yes. Or update affected models.

Reply:
Will trigger a full incremental build from the Jenkins when newer editions are found. This is done by removing the backup files `docs/.jekyll-metadata`, `docs/models.json`, `docs/references.json`, `docs/benchmarking.json`